tests/KeyTest.php

```diff
@@ -22,7 +39,24 @@ public function testList()
     public function testGet()
     {
         $client = $this->getClient(self::objectClass);
-        $api = $client->get(['id' => 'abc123']);
+        $mockBody = '{
```

Comment:
I'm curious: why the switch to just using mocked values? It looks like the previous version of the test(s) made actual API calls, though I could be mis-reading it.

Reply:
It looked to me that previoiusly it was using the mock values in (for example)
https://github.com/silinternational/apiaxle-sdk-php/blob/master/src/mock/key.php
which would be consumed by the findMock function in 
https://github.com/silinternational/apiaxle-sdk-php/blob/master/src/mock/MockResponse.php

Reply:
And the mock middleware would have been keeping the actual api call from happening.
https://github.com/silinternational/apiaxle-sdk-php/blob/master/src/middleware/MockSubscriber.php

Reply:
Okay, thanks